src/routes/legislative-process/+page.svelte

```diff
@@ -100,6 +102,53 @@
 			'ร่างแก้ไขบทบัญญัติใน รธน.': '50,000 คน'
 		}
 	];
+
+	let overallElement1: HTMLElement,
+		overallElement2: HTMLElement,
+		overallElement3: HTMLElement,
+		overallElement4: HTMLElement,
+		overallElement5: HTMLElement;
+	let legislationProcessElement1: HTMLElement, legislationProcessElement2: HTMLElement;
+	let politicianVotingElement1: HTMLElement,
+		politicianVotingElement2: HTMLElement,
+		politicianVotingElement3: HTMLElement,
+		politicianVotingElement4: HTMLElement;
+	let politicianActionElement1: HTMLElement,
+		politicianActionElement2: HTMLElement,
+		politicianActionElement3: HTMLElement;
+	let currentNavElementId: string;
+	onMount(() => {
+		if (window.matchMedia('(min-width: 672px)').matches) {
+			const scroller = scrollama();
+
+			scroller
+				.setup({
+					step: [
```

Comment:
`bind:this` with a variable is usually a recommended way because it is declarative, but introducing 14 variables for each section is a bit overwhelming.

I think you can `bind:this` to the smallest element that contains all the `h2`, then just use `querySelectorAll` on `h2` elements like

```ts
scroller
		.setup({
			  step: bodyContainer.querySelectorAll('h2')
```

We avoid using `document` to avoid selecting every `h2` in the global scope.

**NOTE:**
- I found an invalid `h2` element, "สถานะของกฎหมายในเว็บไซต์นี้ แบ่งได้เป็น 3 ประเภท" should be `h3`, please help fixing that.

Reply:
Ah.. so that's how to do it. Nice!